identity/src/androidTest/java/com/android/identity/IdentityCredentialTest.java

```diff
@@ -98,6 +95,7 @@ private void checkStaticAuthData(int numAuthKeys, int staticAuthDataSizeBytes)
       try {
         store.getCredentialByName(CREDENTIAL_NAME, CIPHER_SUITE);
       } catch (RuntimeException e) {
+        e.printStackTrace();
         fail(String.format(Locale.US,
                 "Failed to load credential for %d authKeys with %d bytes static auth data each",
                 numAuthKeys, staticAuthDataSizeBytes));
```

Comment:
Please add some additional tests for things like corrupted magic, invalid input length, etc.